File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1155.cpp
#include <iostream>

using namespace std;

int main(){
    double soma=0, divisao, i;
    for(i=1;i<=100;i++){
        divisao=1/i;
        soma+=divisao;
    }
    cout.precision(2);
    cout.setf(ios::fixed);
    cout << soma << endl;

    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1155.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner ler = new Scanner(System.in);
        
        double soma=0, divisao, i;
        for(i=1;i<=100;i++){
            divisao=1/i;
            soma+=divisao;
        }
    	System.out.printf("%.2f\n", soma);

    }
}